CMakeLists.txt

```diff
@@ -160,7 +162,7 @@ if(ENABLE_DEEPKS)
   include(FetchContent)
     FetchContent_Declare(
       libnpy
-      GIT_REPOSITORY https://github.com/llohse/libnpy.git
+      GIT_REPOSITORY https://github.com.cnpmjs.org/llohse/libnpy.git
```

Comment:
it would be better not to use the mirror in the online repository :D but I'll change it back so no worry